src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -108,7 +119,11 @@ private void setAccelerator(MenuItem menuItem, KeyCombination keyCombination) {
      */
     void fillInnerParts() {
         personListPanel = new PersonListPanel(logic.getFilteredPersonList());
-        personListPanelPlaceholder.getChildren().add(personListPanel.getRoot());
+        studentListPanelPlaceholder.getChildren().add(new PersonListPanel(logic.getFilteredPersonList()).getRoot());
+        moduleListPanelPlaceholder.getChildren().add(new PersonListPanel(logic.getFilteredPersonList()).getRoot());
+        groupListPanelPlaceholder.getChildren().add(new PersonListPanel(logic.getFilteredPersonList()).getRoot());
+        sessionListPanel = new SessionListPanel(logic.getFilteredSessionList());
```

Comment:
also a little unsure of the fxml this part, might need @PotatoCombat help

Reply:
Here you are placing a ListPanel inside the area designated by the placeholder.

Basically you are "uprooting" the ListPanel, and "planting" it into the placeholder.

PersonListPanel here just takes in an observeable list.